src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/helper/reports/zipfilesummary/ZipFileSummaryItem.java

```diff
@@ -15,7 +15,7 @@
     private final String classification;
     private final String ccdId;
     private final String ccdAction;
-    private final String envelopeId;
+    private final String documentControlNumber;
```

Comment:
there may be more than one payment record

Reply:
Wouldn't there be another record for another payment record. We are mapping it with dcn. If we have more than one payment record, there will different records for them If that makes sense?

Reply:
in one envelope there might be more than 1 payment and you are search for envelope.

Reply:
Yes right but for second payment we will have second record with same envelopeID

Reply:
if it is the case it is completely wrong. You are counting the returned values and adding counts etc.